web/src/integTest/java/com/microsoft/applicationinsights/azuretest/CompareTelemetryWithAzureExportTests.java

```diff
@@ -0,0 +1,342 @@
+package com.microsoft.applicationinsights.azuretest;
```

Comment:
The package name should reflect the what's tested (spring app?) and not how this test is done (using azure).
Please review the names in the UT to find a more appropriate package name (note that changing the package name probably also implies moving the files to a different folder)